handy/poller.cc

```diff
@@ -42,9 +44,12 @@ PollerEpoll::PollerEpoll() {
 
 PollerEpoll::~PollerEpoll() {
     info("destroying poller %d", fd_);
-    while (liveChannels_.size()) {
```

Comment:
a connection's close may close another connection, so it may remove channel from liveChannel, may cause for loop fail

Reply:
Ok, let it keep the status quo, but it doesn't make sense in my view.